tests/network/hotplug.go

```diff
@@ -304,9 +248,81 @@ var _ = SIGDescribe("nic-hotplug", decorators.InPlaceHotplugNICs, func() {
 	})
 })
 
+func createVmiWithHotpluggedInterface(plugMethod hotplugMethod) *v1.VirtualMachineInstance {
```

Comment:
The interface cannot be hotplugged without the NAD, so I see the NAD creation and part of "WithHotpluggedInterface".
I prefer not to take out the NAD creation since all the tests that invoke `createVmiWithHotpluggedInterface` will also invoke the NAD creation method. The two methods won't have existence one without the other.

There is indeed a test that reuses the NAD, but this test won't invoke the NAD creation method separately, it is just reusing the NAD the was created for the hotplugged interface.

Reply:
@AlonaKaplan WDYT about placing NAD creation in `BeforeEach` section?

Reply:
I can try:)